www/Dockerfile

```diff
@@ -1,11 +1,9 @@
-FROM nginx:1.11-alpine
-
-# Add bash for our entrypoint script
-RUN apk add --no-cache bash
+FROM nginx:1.11
```

Comment:
Breaks HTTP2, see https://github.com/desec-io/desec-www/commit/a892c415b1f1dccd14883c7ff3840c347a7440f0

Reply:
Then we can just undo this change and install openssl :+1: